app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,21 @@
+class TagsController < ApplicationController
+  TAGGABLE_MODELS = %w(Group Resource).freeze
```

Comment:
Yes, once this PR and #120 are merged, it will be super easy to make lists taggable. I can do it when we implement the pages to navigate lists.